tests/tracer/test_writer.py

```diff
@@ -411,7 +411,7 @@ def endpoint_test_reset_server():
 
 
 def test_flush_connection_timeout_connect():
-    writer = AgentWriter(_HOST, 2019)
+    writer = AgentWriter("http://%s:%s" % (_HOST, 2019))
```

Comment:
Might be nitpicking since there's only one arg being passed in, but it would be more readable if the agent_url was explicitly called as a keyword arg following the AgentWriter.__init__ doc.